dev/com.ibm.ws.concurrent.persistent_fat_multiple/bnd.bnd

```diff
@@ -0,0 +1,40 @@
+#*******************************************************************************
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
+
+src: \
+	fat/src,\
+	test-applications/persistmultitest/src
+
+fat.project: true
+fat.test.databases: true
+
+-buildpath: \
+	com.ibm.websphere.javaee.annotation.1.1;version=latest,\
+	com.ibm.websphere.javaee.concurrent.1.0;version=latest,\
+	com.ibm.websphere.javaee.servlet.3.1;version=latest,\
+	com.ibm.websphere.javaee.transaction.1.1;version=latest,\
+	com.ibm.ws.concurrent.persistent;version=latest,\
+	org.testcontainers:database-commons;version=1.12.0,\
+	org.testcontainers:jdbc;version=1.12.0,\
+	org.testcontainers:testcontainers;version=1.12.0,\
+	org.testcontainers:db2;version=1.12.0,\
```

Comment:
We do need need these compile time dependencies since the factory class calls:
```java
return new Db2Container().acceptLicense()
.withStartupTimeout(Duration.ofMinutes(FATRunner.FAT_TEST_LOCALRUN ? 5 : 15))
.withLogConsumer(DatabaseContainerFactory::log);
```
which requires us to `import org.testcontainers.containers.Db2Container;`

When we move the factory class to `fattest.simplify`, then I agree we will just need the generic jdbc module